### PR TITLE
Examples: Clean up

### PR DESCRIPTION
**Pull request by Mugen87**
`dat.gui.min.js`, `gunzip.min.js`, `mmdparser.min.js` and `stats.min.js` are not used in the repo anymore.

### CODE REVIEW

File: examples/models/obj/verify/verify.html [hunk 1]
```diff
@@ -71,9 +71,6 @@
 			<div id="feedback"></div>
 		</div>
 
-		<script src="../../../js/WebGL.js"></script>
-		<script src="../../../js/libs/dat.gui.min.js"></script>
```
Comment (Mugen87) [inline examples/models/obj/verify/verify.html]: These imports were unnecessary...

**Comment by mrdoob** [comment]
Thanks!

**Comment by WestLangley** [comment]
But they are used by me, and I bet other users, too. Also SO fiddles...

At least revert the removal of dat.gui.min.js and stats.min.js, please.

**Comment by Mugen87** [comment]
Sorry, but I think you and others can include the libraries from elsewhere. The repository is not intended as part of a CDN. Directories and project structures change over time and some amount of breakage is inevitable.

**Comment by mrdoob** [comment]
I think keeping `dat.gui.min.js` and `stats.min.js` for some months is fair 👌

**Comment by Mugen87** [comment]
Okay, agreed. I'll add the files tomorrow back.

**Pull request merged**